base_classes/NXdata.nxdl.xml

```diff
@@ -141,8 +142,8 @@
 		   field name in the :ref:`axes &lt;/NXdata@axes-attribute&gt;` attribute are used.
 
 		4. When :ref:`AXISNAME_indices &lt;/NXdata@AXISNAME_indices-attribute&gt;` is the same as the indices of "AXISNAME" in the
-		   :ref:`axes &lt;/NXdata@axes-attribute&gt;` attribute, there is no need to provide
-		   :ref:`AXISNAME_indices &lt;/NXdata@AXISNAME_indices-attribute&gt;`.
+		   :ref:`axes &lt;/NXdata@axes-attribute&gt;` attribute and :ref:`AXISNAME &lt;/NXdata/AXISNAME-field&gt;` is one-dimensional,
+		   the :ref:`AXISNAME_indices &lt;/NXdata@AXISNAME_indices-attribute&gt;` attribute can be omitted.
```

Comment:
Addresses @PeterC-DLS comment https://github.com/nexusformat/definitions/pull/1396#discussion_r1722968603

Reply:
For reference: for higher dimensional axes the AXISNAME_indices attribute can never be omitted unambiguously by using the indices in `@axes` instead. For example `@axes=["x", "x", "z"]` could be `@x_indices=[0,1]` or `@x_indices=[1,0]`.

Reply:
Sure it can be omitted for nD if it can be derived "unambigously". But then to define what "unambigously" means in this context would be rather convoluted. For example it won't work when `signal[23,23,17]`.

 So your original suggestion to omit AXISNAME_indices only for 1D still makes sense imo.